internal/entities/liquidity_provider/configuration.go

```diff
@@ -47,7 +48,7 @@ type HashedCredentials struct {
 }
 
 type ConfigurationType interface {
-	PeginConfiguration | PegoutConfiguration | GeneralConfiguration | HashedCredentials
+	PeginConfiguration | PegoutConfiguration | GeneralConfiguration | HashedCredentials | TrustedAccountDetails
```

Comment:
This is ok, but if you decide to count a trusted account as configuration then you don't need to implement a separate signature function